### PR TITLE
Move settings to main webpack

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -30,23 +30,30 @@
...
     "v-tooltip": "^2.0.0-rc.33",
-    "vue-click-outside": "^1.0.7"
+    "vue": "^2.6.6",
+    "vue-click-outside": "^1.0.7",
+    "vue-infinite-loading": "^2.4.3",
+    "vue-localstorage": "^0.6.2",
+    "vue-multiselect": "^2.1.3",
+    "vue-router": "^3.0.2",
+    "vuex": "^3.1.0",
+    "vuex-router-sync": "^5.0.0"
```
Comment (juliushaertl) [inline package.json]: nextcloud-password-confirmation seems missing:

```
    ERROR in ./settings/src/store/api.js
    Module not found: Error: Can't resolve 'nextcloud-password-confirmation' in '/drone/src/github.com/nextcloud/server/settings/src/store'
     @ ./settings/src/store/api.js 23:0-62 62:11-26
     @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./settings/src/views/Users.vue?vue&type=script&lang=js&
     @ ./settings/src/views/Users.vue?vue&type=script&lang=js&
     @ ./settings/src/views/Users.vue
     @ ./settings/src/router.js
@ ./settings/src/main-apps-users-management.js
```

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rullzer) [reply package.json]: good catch. Added.

**Comment by rullzer** [comment]
Alsoupdated drone.yml to remove the obsolete jobs

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Comment by MorrisJobke** [comment]
Webpack build fails with 

```
Uncommited changes in webpack build
```

**Comment by rullzer** [comment]
Weird for me it is clean.... @juliushaertl any clue?

**Comment by juliushaertl** [comment]
@rullzer Maybe still an unused node_modules folder somewhere? I pushed a new build.

**Comment by rullzer** [comment]
Yes you are right

**Pull request merged**